src/styles/card.scss

```diff
@@ -0,0 +1,135 @@
+$paddingY: 50px;
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
+  width: 1000px;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  justify-content: center;
+  padding: $paddingY 0;
+}
+
+.card-apple {
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  margin: 0 24px 46px;
+
+  &--width {
+    width: 200px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(5),
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &__img {
+    display: block;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+    width: 100%;
+  }
+
+  &__commodity {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+
+    color: #616070;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+  }
+
+  &__feedback {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    display: flex;
+    margin-left: 17px;
+    align-items: center;
+  }
+
+  &__price {
+    font-family: inherit;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__world {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: #616070;
+
+  }
+
+  &__number {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+
+  }
+}
+
+.buttom {
+  display: flex;
+  margin: 0;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  justify-content: center;
+  text-transform: uppercase;
+  border: 1px solid #00acdc;
+
+  text-decoration: none;
+  color: white;
+
+  align-items: center;
+  font-size: 14px;
+  font-weight: 700;
+
+  line-height: 16px;
+  text-align: center;
+
+  &--hover:hover {
+    box-sizing: border-box;
+    background-color: #fff;
+    transition: 0.5s;
```

Comment:
Move transition to the base class (not for hover state) and specify the properties that should be changeable